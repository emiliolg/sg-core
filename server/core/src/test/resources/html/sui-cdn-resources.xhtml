<div sui-view="tekgenesis.service.html.Views">
    <img src="@Resource(img/image.jpeg)"/>
    <img src="@Sha(img/image.jpeg)"/>
</div>