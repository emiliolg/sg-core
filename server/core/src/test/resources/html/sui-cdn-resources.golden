<!DOCTYPE html>
<div>
    <img src="http://cdnd.epi.ninja/sha/img/image.jpeg/c4bfa393f1"></img>
    <img src="http://cdnd.epi.ninja/public/img/image.jpeg"></img>
</div>